{
    "uuid": "5723d4a4-f63d-4184-a110-58ba3509434c",
    "historyId": "ОбщийМодуль.СА_ПроцедурыСканАрхиваКлиент.Модуль. 446,3: Процедура или функция с указанным именем не определена",
    "name": "ОбщийМодуль.СА_ПроцедурыСканАрхиваКлиент.Модуль. 446,3: Процедура или функция с указанным именем не определена",
    "fullName": "ОбщийМодуль.СА_ПроцедурыСканАрхиваКлиент.Модуль. 446,3: Процедура или функция с указанным именем не определена",
    "start": 1661424461326,
    "stop": 1661424483037,
    "statusDetails": {
        "known": false,
        "muted": false,
        "flaky": false
    },
    "status": "failed",
    "stage": "finished",
    "steps": [],
    "parameters": [],
    "labels": [
        {
            "name": "package",
            "value": "ОбщийМодуль.СА_ПроцедурыСканАрхиваКлиент.Модуль"
        },
        {
            "name": "story",
            "value": "Процедура или функция с указанным именем не определена"
        },
        {
            "name": "feature",
            "value": "Синтаксическая проверка конфигурации. Конфигуратор"
        },
        {
            "name": "host",
            "value": "Синтаксическая проверка конфигурации. Конфигуратор"
        }
    ],
    "links": [],
    "attachments": [],
    "description": "Строка 446,3: Процедура или функция с указанным именем не определена (Предупреждение) #> <<?>>Предупреждение(мОписаниеОшибки); (Проверка: Внешнее соединение)"
}
